spec/system/admin/enterprises/terms_and_conditions_spec.rb

```diff
@@ -31,7 +31,7 @@ def go_to_business_details
         go_to_business_details
 
         # Add PDF
-        attach_file "enterprise[terms_and_conditions]", original_terms
+        attach_file "enterprise[terms_and_conditions]", original_terms, make_visible: true
```

Comment:
Cool, I didn't know `make_visible`. That's handy. But it also means that we are not testing the new functionality, right?

Reply:
Yes, we don't test the full functionality. Actually, it's impossible because once the system file browser opens there is no way for the driver to interact with it.